src/prefect/cli/events.py

```diff
@@ -0,0 +1,81 @@
+import asyncio
+from enum import Enum
+
+import orjson
+import typer
+import websockets
+from anyio import open_file
+
+from prefect.cli._types import PrefectTyper
+from prefect.cli._utilities import exit_with_error
+from prefect.cli.root import app
+from prefect.events.clients import PrefectCloudEventSubscriber
+from prefect.events.filters import EventFilter
+
+events_app = PrefectTyper(name="events", help="Commands for working with events.")
+app.add_typer(events_app, aliases=["event"])
+
+
+class StreamFormat(str, Enum):
+    json = "json"
+    text = "text"
+
+
+@events_app.command()
+async def stream(
+    format: StreamFormat = typer.Option(
+        StreamFormat.json, "--format", help="Output format (json or text)"
+    ),
+    output_file: str = typer.Option(
+        None, "--output-file", help="File to write events to"
+    ),
+    event_filter: str = typer.Option(None, "--event-filter", help="Event filter"),
```

Comment:
Just for future reference, the filter for audit log events on the account-level stream would be:

```
--event-filter '{"related": {"role": "actor"}}'
```

Reply:
Going to add filters in later